cmd/dcrdata/public/scss/connection.scss

```diff
@@ -6,12 +6,8 @@
     box-shadow: rgb(0 255 0 / 0%) 0 0 0;
   }
 
-  50% {
-    box-shadow: rgb(0 255 0 / 100%) 0 0 10px;
-  }
-
   to {
-    box-shadow: rgb(0 255 0 / 0%) 0 0 0;
+    box-shadow: rgb(0 255 0 / 100%) 0 0 10px;
```

Comment:
update to `keyframe-selector-notation rule` require `from`: 0% and `to`: 100%(or anything but not 0%)